src/services/0_ExternToolsMS/logic/EtherpadClient.js

```diff
@@ -78,70 +77,70 @@ class EtherpadClient {
 		return rp(this.createSettings({
 			endpoint: 'createAuthorIfNotExistsFor',
 		}, params))
-		.then((res) => this.handleEtherpadResponse(res))
-		.catch((err) => { this.handleEtherpadError(err, this.err.createOrGetAuthor) } );
+			.then((res) => this.handleEtherpadResponse(res))
+			.catch((err) => { this.handleEtherpadError(err, this.err.createOrGetAuthor) } );
 	}
 
 	createOrGetGroup(params) {
 		return rp(this.createSettings({
 			endpoint: 'createGroupIfNotExistsFor',
 		}, params))
-		.then((res) => this.handleEtherpadResponse(res))
-		.catch((err) => { this.handleEtherpadError(err, this.err.createOrGetGroup) } );
+			.then((res) => this.handleEtherpadResponse(res))
+			.catch((err) => { this.handleEtherpadError(err, this.err.createOrGetGroup) } );
```

Comment:
Codacy found an issue: [There should be no space before this paren.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045798)